class.photon.php

```diff
@@ -422,9 +421,30 @@ public function filter_image_downsize( $image, $attachment_id, $size ) {
 			 * 	 @type $size Image size. Can be a string (name of the image size, e.g. full) or an integer.
 			 * }
 			 */
-			apply_filters( 'jetpack_photon_override_image_downsize', false, compact( 'image', 'attachment_id', 'size' ) )
-		)
+			false === apply_filters( 'jetpack_photon_admin_allow_image_downsize', false, compact( 'image', 'attachment_id', 'size' ) )
```

Comment:
I think that's fine, as long as the VIP Go knows to warn their clients about it whenever they want to stop using Photon. @mjangda What do you think?